code/modules/GOAP/monkey/actions.dm

```diff
@@ -0,0 +1,350 @@
+/datum/goap_action/monkey/throwitem
+	name = "Throw item"
+	cost = 1
+	cooldown = 20
+
+/datum/goap_action/monkey/throwitem/New()
+	..()
+	preconditions = list()
+	preconditions["grabbedWeapon"] = TRUE
+	preconditions["hasItem"] = TRUE
+	preconditions["hasHumanItem"] = TRUE
+	preconditions["GUN"] = FALSE // don't throw your gun!!
+	effects = list()
+	effects["replaceItem"] = TRUE
+
+/datum/goap_action/monkey/throwitem/AdvancedPreconditions(mob/living/carbon/monkey/C, list/worldstate)
+	var/obj/item/X = locate() in C.held_items
+	if(!X)
+		return FALSE
+	for(var/mob/living/carbon/H in view(5, C))
```

Comment:
why not have monkeys try to throw items at simplemobs as well?

Reply:
good catch, I can type this to living

Reply:
`GLOB.mob_living_list` or `GLOB.alive_mob_list` will work, don't forget checking for distance and seeing if their on the same z level

```suggestion
	for(var/mob/living/carbon/H in GLOB.mob_living_list)
```
```suggestion
	for(var/mob/living/carbon/H in GLOB.alive_mob_list)
```